src/data/polynomial/partial_fractions.lean

```diff
@@ -0,0 +1,132 @@
+/-
+Copyright (c) 2022 The Xena Project. All rights reserved.
```

Comment:
I'm not a copyright lawyer and I'm pretty sure these notices are mostly for show anyway, but I think we should attribute the copyright to a specific (natural or legal) person here.

Reply:
Heh, I wrote that because I wrote the skeleton for this file before I found someone at Xena to do it. Siddarth feel free to change it to your name!